### DISCUSSION TITLE
Welcome to rs-manager Discussions!

### DISCUSSION THREAD

**Discussion by sandiz** [sandiz/rs-manager#114] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JustinAiken** [comment]
👋

**Comment by kevinl7778** [comment]
Hello, I am running windows 10 and I have tried the lates and second to lates versions and both result in a white screen after detecting 6256 songs, I have tried as an admin and not without change also with antivirus off. Any insight yo ucan provide is greartly appreceited.

Kevin

**Comment by kevinl7778** [comment]
I removed my dlc folder and placed it on the desktop and it passed the initial startup but after attempting explor psarc/psarc directory i get the same white screen, is there a number set to max number of songs?

**Reply by sandiz** [reply]
tracking the issue here: https://github.com/sandiz/rs-manager/issues/168 -- please check the comment there and submit a screenshot of the logs. thanks!
